RELEASE.md

```diff
@@ -0,0 +1,38 @@
+# Release Process
+cstor follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/openebs/openebs/projects).
```

Comment:
Link should be: https://github.com/orgs/openebs/projects

Reply:
done

Reply:
for other repo i've drafted PR:
https://github.com/openebs/libcstor/pull/53
https://github.com/openebs/velero-plugin/pull/71
We will push further doc related fixes to above pr and will proceed with merge during RC1.